src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -95,232 +32,19 @@ const NavBar: React.FC<NavBarPropsType> = ({
       : undefined
   );
 
-  const {
-    location: { pathname },
-  } = useHistory();
-  const isOnPlaya = pathname.toLowerCase() === venueInsideUrl(PLAYA_VENUE_ID);
-
-  const now = firebase.firestore.Timestamp.fromDate(new Date());
-  const futureUpcoming =
-    venue?.events?.filter((e) => e.ts_utc.valueOf() > now.valueOf()) ?? []; //@debt typing does this exist?
-
-  const hasUpcomingEvents = futureUpcoming && futureUpcoming.length > 0;
```

Comment:
Where did this move to and/or why was it removed?

Reply:
This was used for the Tickets modal for burning man but it's no longer being used, I decided to remove them because they are not in the current UI.

Reply:
Cool, makes sense